MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+"bazel-contrib/rules_oci"
+
+module(
+    name = "contrib_rules_oci",
+    compatibility_level = 1,
+    version = "0.0.0",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.3.0")
+bazel_dep(name = "aspect_bazel_lib", version = "1.24.2")
```

Comment:
Version 1.28.0 is available as of 5 March 2023, per bazelbuild/bazel-central-registry#487.